runway/hooks/staticsite/auth_at_edge/domain_updater.py

```diff
@@ -0,0 +1,81 @@
+"""User Pool Client Domain Updater."""
+import logging
+from typing import Any, Dict, Optional, Union  # pylint: disable=unused-import
+
+from runway.cfngin.session_cache import get_session
+from runway.cfngin.providers.base import BaseProvider  # pylint: disable=unused-import
+
+LOGGER = logging.getLogger(__name__)
+
+
+# pylint: disable=unused-argument
+def update(context,  # type: context.Context
+           provider,  # type: BaseProvider
+           **kwargs  # type: Optional[Dict[str, Any]]
```

Comment:
This is just style & can be resolved & deferred for now, but since these are python 3+ only functions it would probably be better to not use the py2-compatible type hint syntax (i.e. instead do `def update(context: context.Context,` so the unused-import disabling can be eliminated

Reply:
Are you able to do py3 syntax when you have to support py2? I honestly didn't even know you could do that or I would have went with py3

Reply:
You might actually be correct.

What I was going to say was: these function zips won't ever be executed outside of the Lambda execution environment (where we control which version of python is used), so we don't need to maintain the sample compatibility (in contrast to somewhere like `runway/module/staticsite.py` which might still be executed on a py2 system).

That said, if leaving out the py3 syntax makes py2 runway installs able to build the function zips, then I'm fine leaving them as is.